flows/msg.go

```diff
@@ -168,11 +168,40 @@ func (m *MsgOut) Locale() i18n.Locale { return m.Locale_ }
 // UnsendableReason returns the reason this message can't be sent (if any)
 func (m *MsgOut) UnsendableReason() UnsendableReason { return m.UnsendableReason_ }
 
+// TemplateParamUUID is the UUID of a param
+type TemplateParamUUID uuids.UUID
+
+// TemplateParam represents a single parameter
+type TemplateParam struct {
+	Type_  string            `json:"type"`
```

Comment:
Do we need `type` here? Wondering if each `Msg.Templating` could be stored as something like...

```go
type ComponentVariables struct {
    UUID uuids.UUID
    Variables []string
}

map[string]ComponentVariables
```

and then the localization can be of the `ComponentVariables` instead of each param.

Reply:
I think using `Variables []string` we are loosing the type for the parameter/variable and that is essential to know I think

Reply:
Well let's really think... do we need it? Where?

Reply:
For example when we have the expected parameters as an image, video or document attachment

https://github.com/nyaruka/goflow/pull/1199/files#diff-f146c45610389d4ec808d1c8edafab8ca680779f7da5a7393ba3e8c3b86f427aR158